packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -199,19 +199,28 @@ await checkpointSync.execution.job.stop();
 
 // Unknown block sync
 // ========================================================
+const headForUnknownBlockSync = await env.nodes[0].beacon.api.beacon.getBlockV2("head");
+ApiError.assert(headForUnknownBlockSync);
 const unknownBlockSync = await env.createNodePair({
   id: "unknown-block-sync-node",
   beacon: {
     type: BeaconClient.Lodestar,
-    options: {clientOptions: {"network.allowPublishToZeroPeers": true, "sync.disableRangeSync": true}},
+    options: {
+      clientOptions: {
+        "network.allowPublishToZeroPeers": true,
+        "sync.disableRangeSync": true,
+        // Range sync node start when other nodes are multiple epoch behind and
+        // range sync triggers only if the gap is maximum `slotImportTolerance * 2`
+        // so a lot of times sim tests timeout unknownBlockSync node does not sync up
```

Comment:
It is not clear to me what this means. Why does the node range sync if other nodes are behind, isn't range sync triggered if the node itself is behind? (based on clocks slot vs head slot)